WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -73,16 +80,19 @@ extension ReaderTabView {
         tabBar.addTarget(self, action: #selector(selectedTabDidChange(_:)), for: .valueChanged)
 
         viewModel.fetchReaderMenu() { [weak self] items in
-            guard let items = items, let self = self else {
+            guard let items = items, items.count > 0, let self = self else {
```

Comment:
Could probably shorten this to `guard items?.isEmpty() == true` or use `isEmpty()` here.

Reply:
this is gonna change a bit in the new PR, please bear with me :)